bookkeeper-dist/all/pom.xml

```diff
@@ -152,7 +152,13 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-deploy-plugin</artifactId>
-        <version>${maven-deploy-plugin.version}</version>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-install-plugin</artifactId>
```

Comment:
This wounds weird

Reply:
Thanks for the suggestions on Slack @eolivelli . I have resolved this by reverting the changes to skip install and instead changed packaging from jar to pom for `bookkeeper-dist/all`, `bookkeeper-dist/bkctl` and `bookkeeper-dist/server`.